archive_upload/handlers/dsmc_handlers.py

```diff
@@ -653,19 +654,31 @@ def post(self, runfolder):
             os.path.join(path_to_archive_root, runfolder) + "_archive")
 
         exclude_dirs = self.config["exclude_dirs"]
-        exclude_extensions = self.config["exclude_extensions"]
+        exclude_extensions = []
```

Comment:
Maybe the default could be to exclude `*bcl` files to avoid archiving both `fastq` and `bcl` files. 

Worth noting: there is a newer bcl format called `cbcl` that (at least) the NovaSeqs outputs. I thought we had adapted this service for them, but I guess it wasn't needed since the `Data` folder always has been excluded.

Reply:
The code has been updated with the new approach we discussed (see updated PR description). We now support both config files (ensuring existing systems continue to archive as previously) and overrides via POST parameter. Sending empty arrays for both exclude_extensions and exclude_dirs from the olink workflow will ensure that everything in the Data folder is archived (including bcl/cbcl)